src/data/full-quiz.js

```diff
@@ -6098,5 +6098,14 @@ const fullQuiz = [
     Explanation: "You can use a promise inside a function with the following syntax: return new Promise((resolve, reject) => {})",
     Link: "https://www.freecodecamp.org/news/javascript-promises-explained/",
   },
+  {
+    Question: "What is the correct syntax for "if A is equal to B" ?",

```

Comment:
I would change this to mention strict equality. 

Proposed change: 
What is the correct syntax to check if two variables are strictly equal?

Reply:
hi @jdwilkin4 thank you for replying me, your suggestion are very good but practically how do I apply them? 
(I'm new to this contribution world so I'm learning how to fix issues and create pull requests)
do I have to create another pull request or can I edit the one I did before ? (I can't see where to do it)
I'm sorry to ask for this things that for sure are easy to you but I'm new here

Reply:
Whenever you need to make changes to a PR, you will go back to your local development where your copy of the project is, make those changes and push up a new commit to the branch you are working on. So the branch you are working on is `eliazonta:path-1`. 

Then the PR will be updated with those latest changes and I can review them. 

Hope that makes sense! 😄